src/store/hotspotDetails/hotspotDetailsSlice.ts

```diff
@@ -40,16 +40,18 @@ export const fetchHotspotChallengeSums = async (
     bucket,
   })
   challengeSums.reverse()
-  const totalSum = challengeSums.reduce(
-    (a, b) => ({ sum: a.sum + b.sum } as any),
-  )
-  const prevSum = challengeSumsPrevious.reduce(
-    (a, b) => ({ sum: a.sum + b.sum } as any),
-  )
+  let totalSum = 0
+  let prevSum = 0
+  challengeSums.forEach((sum) => {
```

Comment:
I think this would be a little more concise way of writing this
```ts
const totalSum = challengeSums.reduce((total, { sum }) => total + sum, 0)
```

Reply:
updated